lib/multi_password.rb

```diff
@@ -8,7 +8,12 @@ class MultiPassword
   extend Dry::Configurable
 
   setting :default_algorithm
-  setting :default_options, {}
+  setting :default_options do |value|
+    strategy = registers.fetch(config.default_algorithm).new
```

Comment:
At this time, `config.default_algorithm` may not be defined yet, so this doesn't work. I think you can overwrite `.configure` to add validation code there.

Reply:
thanks sir, I just updated the PR